### ISSUE TITLE
dim parameter of torch.nn.functional.normalize should accept tuples

### ISSUE THREAD

**Issue by calebrob6** [pytorch/pytorch#70100]
### 🐛 Describe the bug

torch.nn.functional.normalize only accepts ints for the dim parameter, however should work in the multidim case

The following code works as expected, however mypy will fail as `dim` is typed as an int.
```
import torch
import torch.nn.functional as F

F.normalize(torch.rand(2,3,4), dim=(0,1))
```

### Versions

Collecting environment information...
PyTorch version: 1.10.0
Is debug build: False
CUDA used to build PyTorch: 11.2
ROCM used to build PyTorch: N/A

OS: Ubuntu 18.04.5 LTS (x86_64)
GCC version: (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0
Clang version: Could not collect
CMake version: version 3.15.5
Libc version: glibc-2.27

Python version: 3.9.7 | packaged by conda-forge | (default, Sep 29 2021, 19:20:46)  [GCC 9.4.0] (64-bit runtime)
Python platform: Linux-5.4.0-1064-azure-x86_64-with-glibc2.27
Is CUDA available: True
CUDA runtime version: 10.1.243
GPU models and configuration:
GPU 0: Tesla V100-PCIE-16GB
GPU 1: Tesla V100-PCIE-16GB
GPU 2: Tesla V100-PCIE-16GB
GPU 3: Tesla V100-PCIE-16GB

Nvidia driver version: 460.91.03
cuDNN version: /usr/lib/x86_64-linux-gnu/libcudnn.so.7.6.5
HIP runtime version: N/A
MIOpen runtime version: N/A

Versions of relevant libraries:
[pip3] efficientnet-pytorch==0.6.3
[pip3] mypy==0.910
[pip3] mypy-extensions==0.4.3
[pip3] numpy==1.21.3
[pip3] pytorch-lightning==1.5.0
[pip3] segmentation-models-pytorch==0.2.0
[pip3] torch==1.10.0
[pip3] torchmetrics==0.6.0
[pip3] torchvision==0.11.1
[conda] cudatoolkit               11.5.0               h36ae40a_9    conda-forge
[conda] efficientnet-pytorch      0.6.3                    pypi_0    pypi
[conda] libblas                   3.9.0            12_linux64_mkl    conda-forge
[conda] libcblas                  3.9.0            12_linux64_mkl    conda-forge
[conda] liblapack                 3.9.0            12_linux64_mkl    conda-forge
[conda] magma                     2.5.4                h6103c52_2    conda-forge
[conda] mkl                       2021.4.0           h8d4b97c_729    conda-forge
[conda] mypy                      0.910                    pypi_0    pypi
[conda] mypy-extensions           0.4.3                    pypi_0    pypi
[conda] numpy                     1.21.3           py39hdbf815f_1    conda-forge
[conda] pytorch                   1.10.0          cuda112py39h4e14dd4_0    conda-forge
[conda] pytorch-gpu               1.10.0          cuda112py39h0bbbad9_0    conda-forge
[conda] pytorch-lightning         1.5.0                    pypi_0    pypi
[conda] segmentation-models-pytorch 0.2.0                    pypi_0    pypi
[conda] torchmetrics              0.6.0                    pypi_0    pypi
[conda] torchvision               0.11.1                   pypi_0    pypi

**Comment by jbschlosser** [comment]
Thanks for the request! We'd accept a PR updating the `F.normalize()` docs and making the typing changes needed to officially support this.

**Comment by calebrob6** [comment]
Cool, will do!
